src/include/kernel/ext_manifest.h

```diff
@@ -62,11 +62,23 @@ struct ext_man_header {
 
 /* Now define extended manifest elements */
 
+/* Extended manifest elements identificators */
+enum ext_man_elem_type {
+	EXT_MAN_ELEM_FW_VERSION		= 0,
```

Comment:
Why would FW version be an enum ?

Reply:
It's FW version packet ID, designed similar to `enum sof_ipc_ext_data`